receiver/bigipreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,19 @@
+receivers:
+  bigip:
+    endpoint: http://localhost:443
+    username: otelu
+    password: $RABBITMQ_PASSWORD
```

Comment:
```suggestion
    password: $BIGIP_PASSWORD
```

Reply:
@BinaryFissionGames Deleted this file as it was not needed